single_tech_samples/databricks/sample3_cluster_provisioning_and_data_access/README.md

```diff
@@ -0,0 +1,96 @@
+# Provisioning Clusters and Access Data
+
+In this sample we are doing a few things:
+
+1. Upload some sample data into a newly created container
+1. Adding the logged in account as a Storage Blob Data Contributor to enable passthrough access
+1. Deploy a default Databricks cluster and a High-Concurrency Databricks cluster
+1. Accessing data using the default cluster by mounting the Data Lake using the Account Key
+1. Accessing data using the high-concurrency cluster by mounting the Data Lake using AD Credential Passthrough
+
+## Prerequisites
+
+* You need to have already deployed the Databricks workspace using sample1
```

Comment:
probably worths mentioning that you'll need to have rights to assign roles as well